tests/CatalogTests/RetryWithExponentialBackoffTests.cs

```diff
@@ -74,6 +74,8 @@ public static IEnumerable<object[]> ReturnsFalseIfResponseStatusBelow500OrWhitel
                 }
             }
 
+            [Theory]
+            [MemberData(nameof(ReturnsFalseIfResponseStatusBelow500OrWhitelisted_Data))]
```

Comment:
@scottbommarito, check here and below that `MemberData` is the expected one.

Reply:
yup